.github/workflows/publish-snap.yaml

```diff
@@ -0,0 +1,26 @@
+name: publish-snap
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-snap:
```

Comment:
Could we put this job into `release.yaml`? We can encapsulate all tasks on release there

Reply:
Yes, but we should wait probably get this pipeline to run green once first.

Currently this pipeline is configured to publish into the `edge` channel and the 'snapcraft.yaml` configured with `grade` set to `devel`. We can move this later into `release.yml` when we set `grade` to `stable` we publish into the `stable` or `candidate` channels instead of the `edge` channel.

Reply:
Good point